nucliadb/nucliadb/search/predict.py

```diff
@@ -392,3 +400,24 @@ async def _iter_answer_chunks(gen):
                 buffer = b""
 
     return _iter_answer_chunks(response.content.iter_chunks())
+
+
+async def _parse_rephrase_response(
+    resp: aiohttp.ClientResponse,
+) -> str:
+    """
+    Predict api is returning a json payload that is a string with the following format:
+    <rephrased_query><status_code>
+    where status_code is "0" for success, "-1" for error and "-2" for no context
+    it will raise an exception if the status code is not 0
+    """
+    content = await resp.json()
```

Comment:
nit if it's a json payload containing just a string with no mapping or list structure, you can probably use the body content directly